### PR TITLE
Hurry up scrubling

### PR DESCRIPTION
**Pull request by alex-ameen**

### CODE REVIEW

**Comment by alex-ameen** [comment]
Bump

File: main.py [hunk 1]
```diff
@@ -1,6 +1,84 @@
...
+            print('Uhoh! "' + field + '" is not a row in our Score Card!')
+            return None
+
+    def row_pretty(field):
+        field = ScoreRound.decode_row(field)
+        if field in ScoreRound.rows_pretty.keys():
+            return ScoreRound.rows_pretty[field][0].title()
+        return field.title()
+
+    def __init__(self, number=1):
```
Comment (Lucaskabela) [inline main.py:44]: What is the number field going to be used for?  I assume for a player number or something similar

Comment (alex-ameen) [reply main.py:44]: I was intended to be a Round Number. So a game would be constructed of a collection of this class to represent each movement along the way.

Comment (alex-ameen) [reply main.py:44]: It was *

**Comment by Lucaskabela** [comment]
Gotcha, I was envisioning a seperate Turn class to handle the work of an individual turn.  Will orchestrate it with the score.  Thank you for your contributions

**Pull request merged**